moveToWebcompat.js

```diff
@@ -1,12 +1,15 @@
+"use strict";
+
+const WEBCOMPAT_ENDPOINT = "https://webcompat.com/issues/new";
+const BUGZILLA_ORIGIN = window.location.origin;
+const FALLBACK_MESSAGE = `More information is available on ${window.location}`;
+
 const REQUIRED = {
   op_sys: "OS (Categories > Platform > OS)",
-  platform: "Platform (Categories > Platform)",
   url: "URL (References > URL)",
```

Comment:
I have decided to not make Platform a required field since Browser and OS are sufficient enough